lightning/src/util/events.rs

```diff
@@ -389,6 +389,15 @@ pub enum MessageSendEvent {
 		/// The channel_update which should be sent.
 		msg: msgs::ChannelUpdate,
 	},
+	/// Used to indicate that a channel_update should be sent to a single peer.
+	/// This is used, in contrast to [`Self::BroadcastChannelUpdate`], when the channel is a
```

Comment:
nit: rephrase to: In contrast to [Self::BroadcastChannelUpdate], this is used when the channel...